ops/flags/install-certs.yml

```diff
@@ -0,0 +1,37 @@
+- type: replace
+  path: /instance_groups/name=bosh/jobs/5:before
+  value:
+    name: ca_certs
+    release: os-conf
+    properties:
+      certs: ((custom_ca_certs))
+
+- type: replace
```

Comment:
All the uaa stuff below does not seem related to installing custom ca certs

Reply:
Initially I didn't have any of this UAA stuff, and enabling this new flag caused the "uaa" job to fail when doing `bucc up`. These changes were proposed by @ramonskie and adding them fixes the issue with UAA. The changes were suggested based on this issue https://github.com/cloudfoundry/uaa/issues/766

The accompanying discussion is in the #bucc channel on cloudfoundry.slack.com (I believe you saw at least some of it already).